src/features/content-explorer/item-list/ItemList.js

```diff
@@ -5,6 +5,7 @@ import getProp from 'lodash/get';
 
 import Column from '@box/react-virtualized/dist/commonjs/Table/Column';
 import Table from '@box/react-virtualized/dist/commonjs/Table';
+import AutoSizer from '@box/react-virtualized/dist/commonjs/AutoSizer';
```

Comment:
super nit: we could move this import above `Column` so that the `Table` related components are grouped together and sorted.

Reply:
Done.